adage/analyze/search_indexes.py

```diff
@@ -25,11 +25,14 @@ class SampleIndex(indexes.SearchIndex, indexes.Indexable):
     """
     text = indexes.CharField(document=True, use_template=True)
     name = indexes.CharField(model_attr='name')
-    experiments = indexes.CharField(model_attr='experiments__all')
+    experiments = indexes.MultiValueField()
     ml_data_source = indexes.CharField(model_attr='ml_data_source', null=True)
 
     def get_model(self):
         return Sample
 
     def index_queryset(self, using=None):
         return self.get_model().objects
+
+    def prepare_experiments(self, obj):
+        return [e.accession for e in obj.experiments.all()]
```

Comment:
Nice! Shall we include a unit test to make sure this method outputs what we expect it to output?

Reply:
Ohh. Yes, good idea.

- [ ] test new `prepare_experiments` method
- [ ] test to ensure future indexing problems are caught automatically